### PR TITLE
Fixes #1654: Add DefaultAudioFragment to the DefaultAudioActivity

### LINKED ISSUE #1654: Add DefaultAudioFragment to the DefaultAudioActivity
The project follows the approach of using a fragment for each activity but that approach isn't used in `DefaultAudioActivity`  
Add a new fragment to the `DefaultAudioActivity`

### PR DESCRIPTION
**Pull request by MohamedMedhat1998**
## Explanation

- Fixes #1654 
- Added **DefaultAudioFragment** to **DefaultAudioActivity**
## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/src/main/res/layout/default_audio_activity.xml [hunk 1]
```diff
@@ -1,63 +1,5 @@
...
-        android:layout_width="match_parent"
-        android:layout_height="6dp"
-        android:background="@drawable/toolbar_drop_shadow" />
-    </FrameLayout>
-  </androidx.constraintlayout.widget.ConstraintLayout>
-</layout>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+  android:id="@+id/default_audio_fragment_container"
+  android:layout_width="match_parent"
+  android:layout_height="match_parent" />
```
Comment (rt4914) [inline app/src/main/res/layout/default_audio_activity.xml]: Missing EOF

**Review by rt4914 — CHANGES_REQUESTED** [review]
Please have a look at https://github.com/oppia/oppia-android/pull/1577 review and update your PRs accordingly.

File: app/src/main/res/layout/default_audio_activity.xml (hunk 1 above)
Comment (MohamedMedhat1998) [reply app/src/main/res/layout/default_audio_activity.xml]: Done

**Comment by MohamedMedhat1998** [comment]
> Please have a look at #1577 review and update your PRs accordingly.

Done

File: app/src/test/java/org/oppia/app/testing/options/DefaultAudioFragmentTest.kt [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+import org.oppia.app.utility.OrientationChangeAction.Companion.orientationLandscape
+import org.robolectric.annotation.LooperMode
+
+@RunWith(AndroidJUnit4::class)
+class DefaultAudioFragmentTest {
+
+  @Test
+  @LooperMode(LooperMode.Mode.PAUSED)
+  fun testAudioLanguage_changeLanguageToEnglish_changeConfiguration_checkEnglishLanguageIsSelected() { // ktlint-disable max-line-length
+    launch<OptionsActivity>(createDefaultAudioActivityIntent("Hindi")).use {
```
Comment (rt4914) [inline app/src/test/java/org/oppia/app/testing/options/DefaultAudioFragmentTest.kt]: If you are checking the `DefaultAudioFragmentTest ` it should not start with `OptionsActivity`, it should start with `DefaultAudioActivity`

**Review by rt4914 — CHANGES_REQUESTED** [review]
PTAL

File: app/src/test/java/org/oppia/app/testing/options/DefaultAudioFragmentTest.kt (hunk 2 above)
Comment (MohamedMedhat1998) [reply app/src/test/java/org/oppia/app/testing/options/DefaultAudioFragmentTest.kt]: Done

**Review by rt4914 — APPROVED** [review]
LGTM

**Pull request merged**